db/seeds.rb

```diff
@@ -6,18 +6,18 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-plan = Plan.create(plan_name: "Daily Success Plan")
+plan = Plan.create(plan_name: 'Daily Success Plan')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.